FileShares/src/FileSharesWeb/FileShareHealthContributor.cs

```diff
@@ -0,0 +1,62 @@
+using System.Runtime.InteropServices;
+using Steeltoe.Common.HealthChecks;
+
+namespace Steeltoe.Samples.FileSharesWeb;
+
+internal class FileShareHealthContributor : IHealthContributor
+{
+    private const ulong ThresholdInBytes = 10 * 1024 * 1024;
+    public string Id => "fileShareHealthContributor";
+
+    public Task<HealthCheckResult?> CheckHealthAsync(CancellationToken cancellationToken)
+    {
+        HealthCheckResult result = Health();
+        return Task.FromResult<HealthCheckResult?>(result);
+    }
+
+    private static HealthCheckResult Health()
+    {
+        if (!string.IsNullOrEmpty(FileShareHostedService.Location) && Directory.Exists(FileShareHostedService.Location))
```

Comment:
The parameter in `Directory.Exists` is nullable, so the first clause can be removed.